package.json

```diff
@@ -31,7 +31,8 @@
     "@types/mocha": "^5.2.6",
     "@types/node": "^20.5.0",
     "ist": "^1.1.1",
-    "mocha": "^10.2.0"
+    "mocha": "^10.2.0",
+    "ts-node": "^10.9.2"
```

Comment:
I had to add `ts-node` otherwise I couldn't run tests. Not sure if it was just my machine, but whatever.

Reply:
Probably due to https://github.com/lezer-parser/generator/blob/d827142436db3724af4b4dd9162132ca343f34e0/.mocharc.cjs#L4